CHAPTER3-COMMIT CONVENTIONS.md

```diff
@@ -0,0 +1,143 @@
+# Chapter 3 - Commit Message Conventions
+
+These rules are adopted from [the AngularJS commit conventions](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits).
+
+* [Subject line](#subject-line)
+    * [Allowed `<type>`](#allowed-type)
+    * [Allowed `<scope>`](#allowed-scope)
+    * [`<subject>` text](#subject-text)
+* [Message body](#message-body)
+* [Message footer](#message-footer)
+    * [Breaking changes](#breaking-changes)
+    * [Referencing issues](#referencing-issues)
+* [Examples](#examples)
+
+
+The format of a commit message should be as follows:
+
+```
+<type>(<scope>): <subject>
+<BLANK LINE>
+<body>
+<BLANK LINE>
+<footer> (optional)
+```
+
+Any line of the commit message should not exceed 100 characters. This allows the message to be easier to read.
+
+## Subject line        
+The subject line should contain a succinct description of the change with context and should not exceed 50 characters.
+
+### Allowed `<type>`
+- build: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- ci: Changes to CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+- docs: Documentation only changes
+- feat: A new feature
+- fix: A bug fix
+- perf: A code change that improves performance
+- refactor: A code change that neither fixes a bug nor adds a feature
+- style: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- test: Adding missing tests or correcting existing tests
+
+### Allowed `<scope>`
+Scope specifies the module of the code changed by the commit, if applicable.
+
+### `<subject>` text
+* use imperative, present tense: “change” not “changed” nor “changes”
+* don't capitalize first letter
+* no single dot (.) at the end
+
+## Message body
+* use imperative, present tense: “change” not “changed” nor “changes”
+* include motivation for the change and contrasts with previous behavior
+
```

Comment:
Again, capitals.
"Include **the** motivation for the change and **the** contrasts with previous behavior"